### PR TITLE
[Chat] Add tests for MentionPopover rendering

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
# What

Added unit tests for showing and hiding mention popover

# Why

Increate tests coverage for the mention feature

# How Tested

Ran tests on beta and stable flavors

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/InputBoxComponent.test.tsx [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+      <InputBoxComponent
+        inlineChildren={true}
+        textValue={changedValue}
+        placeholderText="Enter a message"
+        onChange={(_, newValue) => {
+          changedValue = newValue ?? '';
+        }}
+        maxLength={1000}
+      >
+        <div></div>
```
Comment (emlynmac) [inline packages/react-components/src/components/InputBoxComponent.test.tsx:25]: I'm curious as to why the div is needed here?

Comment (vhuseinova-msft) [reply packages/react-components/src/components/InputBoxComponent.test.tsx:25]: `InputBoxComponent` has a required children props and can't be used as a self closing tag

File: packages/react-components/src/components/InputBoxComponent.test.tsx [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    {
+      id: '1',
+      displayText: 'Test User1'
+    },
+    {
+      id: 'everyone',
+      displayText: 'Everyone'
+    }
+  ];
+  const trigger = '@';
```
Comment (jimchou-dev) [inline packages/react-components/src/components/InputBoxComponent.test.tsx:52]: Maybe add a test for trigger with other character that is not '@' or with trigger that is more than 1 character eg '@@'

Comment (vhuseinova-msft) [reply packages/react-components/src/components/InputBoxComponent.test.tsx:52]: Added!

**Review by jimchou-dev — APPROVED** [review]
LGTM

**Pull request merged**